scripts/rollup-replace.js

```diff
@@ -1,18 +1,33 @@
-import replace from 'rollup-plugin-replace';
+import replace from 'rollup-plugin-re';
 
-export default function(options) {
+// three modules to find-replace in
+const modules = [
+  'VRControls',
+  'VREffect',
+  'OrbitControls'
+];
 
+export default function(options) {
   return replace(Object.assign({
-    'include': ['node_modules/three/examples/js/**'],
-    'delimiters': ['', ''],
-    'THREE.VRControls =': "var THREE = require('three');var VRControls;\nmodule.exports = VRControls =",
-    'THREE.VRControls': 'VRControls',
+    include: ['node_modules/three/examples/js/**'],
+    patterns: [
+      {transform(code, id) {
+        modules.forEach((m) => {
+          if (!(new RegExp(m)).test(id)) {
+            return;
+          }
+
+          // trun the global modifiction into an import and a local variable definition
+          code = code.replace(`THREE.${m} =`, `import * as THREE from 'three';\nconst ${m} =`);
```

Comment:
const will not be understood in the non-es version, I had to change it to var to be able to compile

Reply:
ahhh ok, I will fix that